tests/snapshots/test_glob__identity_glob_example/test_data/example.txt.snap

```diff
@@ -0,0 +1,7 @@
+---
+source: tests/test_glob.rs
+expression: "std::convert::identity"
```

Comment:
So ideally we have something like `input: example.txt` here.

Reply:
I definitely agree but I have no idea where/how to add that metadata :sweat_smile: